src/pretix/plugins/banktransfer/apps.py

```diff
@@ -38,6 +38,11 @@ class PretixPluginMeta:
         version = version
         description = _("Accept payments from your customers using classical wire transfer methods with your own "
                         "bank account.")
+        settings_links = [
+            ((_("Payment"), _("Bank transfer")), "control:event.settings.payment.provider", {"provider": "banktransfer"}),
+            ((_("Bank transfer"), _("Import bank data")), "plugins:banktransfer:import", {}),
+            ((_("Bank transfer"), _("Export refunds")), "plugins:banktransfer:refunds.list", {}),
```

Comment:
I think we need to decide: Do we want this feature to (a) link to a plugin's *settings*, or (b) to all of a plugins features?

If (a), then we need a better name than settings_links and probably a different icon than fa-cog.

If (b), then these two do not belong here.

Reply:
I think linking to more than just settings would be helpful, so maybe `navigation_links`.
Or even split it into two parts, one for settings and one for other view.

Reply:
navigation_links and keeping it as one sounds fine, just don't have a great idea for an icon right now. :link:  is obvious but also feels a little technical

Reply:
I added a "Go to" button with a compass icon for the non-settings items (see screenshot), what do you think? Merging everything into the "Go to" would be fine for me too, but I like the two buttons even better.

Reply:
I think it's fine like this